README.md

```diff
@@ -12,6 +12,13 @@ Make sure the global composer project is configured properly with your shell.
 Notably, the composer global bin directory needs to be in your path.
 See https://getcomposer.org/doc/03-cli.md#global for instructions on how todo that.
 
+## Adding composer global bin to path on Ubuntu
+
+You should add to your ~/.profile 
+```
+export PATH="$PATH:$COMPOSER_HOME/vendor/bin"
```

Comment:
I agree with @bcremer. Also, here we're only linking to composer documentation where the $PATH thing should be explained already. If you think their section is not clear enough, please open a PR there directly so all composer users can benefit from it.